uvm/hier/usr/lib/python3/dist-packages/tests/test_network.py

```diff
@@ -2422,6 +2423,67 @@ def test_702_remove_snort_files(self):
         #sync settings should cleanup *-snort* extension files
         snort_files = find_files(dir_path, search_string)
         assert len(snort_files) == 0
+    
+    def test_703_remove_unreachable_host(self):
+        """
+        Remove unreachable host
+        """
+        # Backup original configuration file and set a new file with updated configuration
+        untangle_vm_conf_filename = "/usr/share/untangle/conf/untangle-vm.conf"
+        untangle_vm_conf_tmp_filename = f"{untangle_vm_conf_filename}.tmp"
+        untangle_vm_conf_original_filename = f"{untangle_vm_conf_filename}.orig"
+
+        shutil.copyfile(untangle_vm_conf_filename, untangle_vm_conf_original_filename)
+
+        host_cleaner_interval = 5000
+        host_cleaner_max_unreachable = 60000
+        vm_conf = []
+        with open(untangle_vm_conf_filename, "r") as file:
+            for line in file:
+                if line.startswith("host_cleaner_interval="):
+                    line = f"host_cleaner_interval=\"{str(host_cleaner_interval)}\"\n"
+                if line.startswith("host_cleaner_max_unreachable="):
+                    line = f"host_cleaner_max_unreachable=\"{str(host_cleaner_max_unreachable)}\"\n"
+                vm_conf.append(line)
+            file.close()
+
+        with open(untangle_vm_conf_tmp_filename, "w") as file:
+            for line in vm_conf:
+                file.write(line)
+            file.close()
+
+        os.replace(untangle_vm_conf_tmp_filename, untangle_vm_conf_filename)
+        ## Restart uvm
+        uvmContext = global_functions.restart_uvm()
+
+        # Add a host entry which is not part of the network
+        unreachable_entry = {}
+        unreachable_entry['usernameDirectoryConnector'] = 'unrechable_host'
+        unreachable_entry['address'] = "31.0.0.1"
```

Comment:
Is this always unreachable?  If not, I recommend changing to the broadcast address on the current LAN which you would need to calculate for whatever the current LAN network is.

Reply:
@cblaise-untangle Having broadcast address seems safer option.
How about we extract the broadcast address from `ifconfig` like below ?
```
"ifconfig | grep ${IPV4} | awk '{printf $6}'"

Reply:
As per slack discussion with Chris,

> I think you want to look at the settings, pick the first LAN interface and then calculate broadcast from that.  Since it the interface should be tied to a physical device (e.g.,eth1) you should be do similar parsing to what you did with the ifconfig command.  Except look into the ip addr command as I believe we're waiting for ifconfig to be depreciated "any day now".

We are using `ip addr` command to extract broadcast address